File: pl-domain/gender-service/src/main/java/com/polylab/gender/GenderServiceApplication.java
package com.polylab.gender;

import com.jxareas.jxcore.annotations.EnableJxCore;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableJxCore
@SpringBootApplication
public class GenderServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(GenderServiceApplication.class, args);
    }

}

File: pl-domain/gender-service/src/main/java/com/polylab/gender/GenderServiceApplication.kt
package com.polylab.gender;

import com.jxareas.efficax.annotations.EnableEfficax
import com.jxareas.efficax.annotations.EnableNonRestrictiveCorsPolicy
import com.jxareas.efficax.extensions.runApp
import org.springframework.boot.autoconfigure.SpringBootApplication

@EnableEfficax
@EnableNonRestrictiveCorsPolicy
@SpringBootApplication
open class GenderServiceApplication

fun main(vararg args : String) = runApp(GenderServiceApplication::class, *args)
